docs/contributing.md

```diff
@@ -0,0 +1,107 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish
+to make via issue, email, or any other method with the owners of this
+repository before making a change.
```

Comment:
Isnt't "before making a change" redundant? Since at the beginning of the sentence there is the sentence "first discuss the change".